collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -612,6 +611,11 @@ protected void onActivityResult(int requestCode, int resultCode,
             return;
         }
 
+        if (intent == null) {
+            Timber.i("The intent has a null value for requestCode: " + requestCode);
```

Comment:
`Timber.w` will call the `Firbase.logcat()` function ([here](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/java/org/odk/collect/android/application/Collect.java#L298)) and that seems ok to me. 
The documentation of Firebase says that Firebase.logcat() logs the message to both subsequent crash report along with the message and also to the logcat. [source](https://developers.google.com/android/reference/com/google/firebase/crash/FirebaseCrash#logcat(int,java.lang.String,java.lang.String))